src/main/flight/pid.c

```diff
@@ -52,6 +52,10 @@
 
 extern uint8_t motorCount;
 extern float dT;
+extern bool motorLimitReached;
```

Comment:
Using `extern` in .c file is very error prone. It is far too easy to change type of variable and forget to update all `extern` declarations. If variable is public, declare it in header file